### DISCUSSION TITLE
Is there a way to get Inserted text from code completion list?

### DISCUSSION THREAD

**Discussion by prashantvc** [microsoft/vscode-discussions#717] [Extension Development QnA] [answered]
I am trying to get the event or the inserted text (with position) from code completion list in the active editor.

I tried to work with LSP client `onNotification` or `onRequest` but both don't trigger when code completion insert the text.
I don't think server sends any notifications

Any suggestions on how to accomplish this?

**Comment by isidorn** [comment]
Hi Prashant.
I do not think this is currently possible. What exactly is your scenario, what would you like to accomplish in your extension? Thanks!

**Reply by prashantvc** [reply, marked as answer]
I accomplished what I was set out to do. I wanted to automatically trigger code complete suggestions after user inserts a completion item from the list

It's easier if I show:

https://github.com/microsoft/vscode-discussions/assets/328076/06a0cfa7-ca98-453e-a1a3-a1b3a5e855cb

I am executing text editor command from LSP server to auto trigger suggestions. So it's all good now :)

**Reply by isidorn** [reply]
Awesome! Thanks!
